ebos/eclcpgridvanguard.hh

```diff
@@ -276,9 +278,30 @@ protected:
         // equilGrid_being a shallow copy only the global view.
         if (mpiRank == 0)
         {
+            const auto& gridProps = this->eclState(false).get3DProperties(); // this is hit
```

Comment:
please remove the hit comment and let it assert.

Reply:
done